mllib/src/test/java/org/apache/spark/ml/classification/JavaRandomForestClassifierSuite.java

```diff
@@ -81,15 +81,15 @@ public void runDT() {
     for (String featureSubsetStrategy: RandomForestClassifier.supportedFeatureSubsetStrategies()) {
       rf.setFeatureSubsetStrategy(featureSubsetStrategy);
     }
-    String realStrategies[] = {".1", ".10", "0.10", "0.1", "0.9", "1.0"};
+    String[] realStrategies = {".1", ".10", "0.10", "0.1", "0.9", "1.0"};
```

Comment:
According to @andrewor14 , java linter is turned off intentionally due to the overhead of maven.